doc/source/releases/24.0_announce_user.rst

```diff
@@ -0,0 +1,196 @@
+
+My workflows: card view
+
+.. figure:: images/24.0-wf-cardview.png
+   :alt: TODO.
+
+My workflows: list view
+
+.. figure:: images/24.0-wf-listview.png
+   :alt: TODO.
+
+New select component for selecting a large amount of options
+-----------------------------------------------------------
+
+This new component addresses the need of selecting a very large number of options at once. The component features a list based layout which can be filtered using plain-text or a regular expression. Options can be added individually, in bulk via the filter value, or by first highlighting a range. 
+
+The component is fully keyboard accessible. All methods of selection and highlighting work via keyboard. The options are not selectable individually with tab, but can be scrolled through using the arrow-keys. The hover hint adapts when a keyboard is used. 
+
+The size of the table can be increased to allow for seeing more options at once.The options in the list are limited to 1000 for performance reasons, but this can be temporarily increased in steps of 500, when reaching the end of a list.
+
+.. figure:: images/24.0-selectview.png
+   :alt: TODO.
+
+The component can be chosen when multiple options are available; a local preference for which component to choose can be set. If no preference is set, the component will be used if there are more than 15 options selected, or more than 500 options are available.
+
+.. figure:: images/24.0-selectdropdown.png
+   :alt: TODO.
+
+Multiple item drag and drop and keyboard navigation
+-----------------------------------------------------------
+
+This new feature allows users to select multiple history items and drag and drop them into other histories or even tool forms! Users can also navigate their history using the arrow keys; multiple history items can be selected with the Shift + ArrowUp/ArrowDown key combination. 
+
+Drag and drop:
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Keyboard navigation and selection:
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Resource grids have been consolidated into tab views
+-----------------------------------------------------------
+
+Grids for Histories, Visualizations and Pages have been consolidated into tab views.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+UI for "relocating" a dataset to a new object store
```

Comment:
```suggestion
Move datasets between data stores.
```

(we were changing the language on "object store" right? because that wasn't user accessible?)